cloudbuild/python-scripts/requirements.txt

```diff
@@ -0,0 +1,34 @@
+absl-py==2.0.0
+avro==1.11.3
```

Comment:
Let's ensure we're using the same avro version as in the connector source code.

Reply:
Seems like we are using 1.11.1, changed to that.